ports/servoshell/egl/app_state.rs

```diff
@@ -115,6 +115,16 @@ impl ServoDelegate for ServoShellServoDelegate {
 }
 
 impl WebViewDelegate for RunningAppState {
+    fn screen_geometry(&self, webview: WebView) -> Option<ScreenGeometry> {
+        let coord = self.callbacks.coordinates.borrow();
+        let screen_size = DeviceIntSize::new(coord.viewport.size.width, coord.viewport.size.height);
+        Some(ScreenGeometry {
+            size: screen_size,
+            available_size: screen_size,
+            offset: Point2D::zero(),
```

Comment:
The offset should be non-zero, since the xcomponent (servo window) is below the URL bar. Do you happen to know how we can get the offset of the xcomponent on the screen? Can be done in a separate PR too